packages/form-builder/src/components/onboarding/steps/withText.tsx

```diff
@@ -0,0 +1,46 @@
+import {renderToStaticMarkup} from "react-dom/server";
+import Logo from "@givewp/form-builder/components/icons/logo";
+import {Button} from "@wordpress/components";
+import {__} from "@wordpress/i18n";
+
+const TextContent = ({title, description, stepNumber, stepCount, isFirst, isLast}) => {
+    return (
+        <div>
+            {isFirst && (
+                <div style={{display: "flex", justifyContent: 'center', margin: '0 auto var(--givewp-spacing-4)'}}>
+                    <Logo/>
+                </div>
+            )}
+            {!isFirst && !isLast &&(
+                <div style={{display:'flex', backgroundColor: 'var(--givewp-blue-25)', fontSize: '12px',padding:'4px',borderRadius:'2px',justifyContent:'space-between', alignItems: 'center'}}>
+                    <div><strong>Step {stepNumber}</strong> out of {stepCount}</div>
```

Comment:
this needs to be internationalized. and I think `Step {stepNumber} of {stepCount}` is best.

Reply:
Resolved.